### ISSUE TITLE
udemy has changed its policy !

### ISSUE THREAD

**Issue by AliSaberi** [FaisalUmair/udemy-downloader-gui#616]
Hi.
Udemy does no longer allows you to download paid courses even by video grabbers like IDM

**Comment by skamranhaq** [comment]
@FaisalUmair Any hope of resolution of this issue?

**Comment by AliSaberi** [comment]
The solution is on video grabbers ! they must change their methods so that they can grab the videos as before. However, I'm not sure if they're willing to do so.

**Comment by ScrubDonkey** [comment]
> The solution is on video grabbers ! they must change their methods so that they can grab the videos as before. However, I'm not sure if they're willing to do so.

Yeah, the only way to do it now is via OBS browser grabbing. Not going to sit there and do that though.

**Comment by strfr** [comment]
It's working on paid courses too. is it just me?
